### PR TITLE
i288 Investigate PWA Issue

### LINKED ISSUE #288: investigate PWA issue
## Basic Information
**[Give a brief summary of what the issue is about]**
- Not working for everyone
- When it does work, for some it doesnt display the logo on the home page but instead displays a screenshot of the website?

### PR DESCRIPTION
**Pull request by RadioArsenic**
## Change Summary

**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form

**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #288

### CODE REVIEW

File: client/vite.config.ts [hunk 1]
```diff
@@ -30,11 +30,16 @@ export default defineConfig({
...
+            src: 'images/CSF_feet_144_sq_px.png',
+            sizes: '144x144',
+            type: 'image/png'
+          },
           {
             src: 'images/CSF_feet_144_sq_px.png',
             sizes: '144x144',
             type: 'image/png',
-            purpose: 'maskable'
+            purpose: 'any'
```
Comment (RadioArsenic) [inline client/vite.config.ts:42]: Don't we just need the first one to have either no purpose or purpose 'any' and then the following ones to have purpose 'maskable'?

**Comment by ChantelleKerr** [comment]
Can we please get this done? It was working with amy's changes

**Comment by RadioArsenic** [comment]
My bad, I thought Ben was trying to fix the install button not the icon. My changes were just to get the icon working again

**Comment by ChantelleKerr** [comment]
> My bad, I thought Ben was trying to fix the install button not the icon. My changes were just to get the icon working again

It seems like its working on my end except when you navigate to another page and then ome back to it. However if you refresh on the about page it will load the button again

Is this the same for you?

**Comment by RadioArsenic** [comment]
> It seems like its working on my end except when you navigate to another page and then ome back to it. However if you refresh on the about page it will load the button again
> 
> Is this the same for you?

If you're talking about the install button on the about page, that's never worked on any of my devices and still isn't. If you're talking about chrome's install button in the address bar, it takes a second to load at first but then is present across all pages

**Comment by ChantelleKerr** [comment]
> > It seems like its working on my end except when you navigate to another page and then ome back to it. However if you refresh on the about page it will load the button again
> > Is this the same for you?
> 
> If you're talking about the install button on the about page, that's never worked on any of my devices and still isn't. If you're talking about chrome's install button in the address bar, it takes a second to load at first but then is present across all pages

Yeah the install button. Do you have it installed anywhere? Put chrome://apps/ in the address bar and see if its there

**Comment by RadioArsenic** [comment]
> > If you're talking about the install button on the about page, that's never worked on any of my devices and still isn't. If you're talking about chrome's install button in the address bar, it takes a second to load at first but then is present across all pages
> 
> Yeah the install button. Do you have it installed anywhere? Put chrome://apps/ in the address bar and see if its there

Button just appeared! Don't know what changed though, having the same thing, if I navigate to a different page and come back I have to refresh

**Comment by ChantelleKerr** [comment]
> > > If you're talking about the install button on the about page, that's never worked on any of my devices and still isn't. If you're talking about chrome's install button in the address bar, it takes a second to load at first but then is present across all pages
> > 
> > 
> > Yeah the install button. Do you have it installed anywhere? Put chrome://apps/ in the address bar and see if its there
> 
> Button just appeared! Don't know what changed though, having the same thing, if I navigate to a different page and come back I have to refresh

Nice, I dont think theres a way around it tbh

**Review by ChantelleKerr — APPROVED** [review]
Nice

**Pull request merged**